lms/djangoapps/bulk_email/tasks.py

```diff
@@ -173,7 +173,8 @@ def perform_delegate_email_batches(entry_id, course_id, task_input, action_name)
     # Get arguments that will be passed to every subtask.
     targets = email_obj.targets.all()
     global_email_context = _get_course_email_context(course)
-
+    if task_input.get('platform_name', False):
```

Comment:
`False` is excessive here IMO. Since the default `default` value of `dict.get` method is `None` which itself is interpreted as boolean False.
```suggestion
    if task_input.get('platform_name'):
```

Reply:
And IMO it's okay to do this explicitly